site/zenodo_rdm/legacy/serializers/schemas.py

```diff
@@ -63,6 +66,92 @@ def dump_resource_type(self, result, original, **kwargs):
                 result[f"{upload_type}_type"] = resource_type_id.split("-")[-1]
         return result
 
+    def _funder(self, funder):
+        """Serialize RDM funder into Zenodo legacy funder."""
+        legacy_funder = {"name": funder["name"]}
+
+        f_identifiers = {}
+        for identifier in funder.get("identifiers"):
+            scheme = identifier["scheme"]
+            if scheme == "oaf":
+                f_identifiers["oaf"] = identifier["identifier"]
+
+            if scheme == "doi":
+                legacy_funder["doi"] = identifier["identifier"]
+
+        if f_identifiers:
+            legacy_funder["identifiers"] = f_identifiers
+
+        value = funder.get("country")
+        if value:
+            legacy_funder["country"] = value
+
+        return legacy_funder
+
+    def _award(self, award):
+        """Serialize an RDM award into a legacy Zenodo grant."""
+        legacy_grant = {"code": award["number"], "internal_id": award["id"]}
+
+        # if english does not exist default to the first key
+        # FIXME: should default to default locale?
+        try:
+            title = award["title"].get("en", next(iter(award["title"])))
+            legacy_grant["title"] = title
+        except StopIteration:
+            pass
+
+        value = award.get("acronym")
+        if value:
+            legacy_grant["acronym"] = value
+
+        a_identifiers = {}
+        for identifier in award.get("identifiers"):
```

Comment:
We might have to configure [VOCABULARIES_AWARD_SCHEMES](https://github.com/inveniosoftware/invenio-vocabularies/blob/master/invenio_vocabularies/config.py#L58). At the momtne it contains only DOI and URL, however legacy records would contain OAF, pURL and eurepo.